### ISSUE TITLE
Not published to npm registry 

### ISSUE THREAD

**Issue by ericclemmons** [taktakpeops/jest-environment-browserstack#164]
```
❯ yarn add --dev @jest-environment-browserstack/selenium-webdriver -W
yarn add v1.22.4
[1/4] 🔍  Resolving packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/@jest-environment-browserstack%2fselenium-webdriver: Not found".

```

**Comment by taktakpeops** [comment]
Hi,

The “old” version of the module is available.

The new version will published during the day.

For now, you can install it by running “yarn add jest-environment-browserstack”.

I will update you here once published.

**Comment by taktakpeops** [comment]
Hi, sorry for the issue. All packages were published, version 2.0.1

It was tested in the basic-usage example.

I am closing the issue. Feel free to open another one if you find anything else wrong.
